### ISSUE TITLE
[COGNITION] 508-defect-3: Character counts MUST announce properly in IE11 + JAWS

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#15079]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a JAWS screen reader user, I want to hear the character counts announced properly after I have entered data about my Express Care issue. It's beneficial to know if I am nearing the character limit.

## Details

This issue is only happening on IE11 + JAWS 2020. All other screen reader + browser combinations I tested worked as expected. It seems there might be two data points being collated to assemble this one string, based on how it's chunked out in the Chrome Dev Tools. Screenshot attached below.

## Acceptance Criteria

- [ ] IE11 + JAWS announces "255 characters remaining" properly instead of just announcing "255"

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* Win10
* IE11
* JAWS 2020
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-express-care-request/additional-details

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-10-20 at 4 02 32 PM](https://user-images.githubusercontent.com/934879/96644550-daf5f280-12ee-11eb-8b2a-f5e20c7cb291.png)

**Comment by jbalboni** [comment]
@1Copenut This should be on staging now

**Comment by 1Copenut** [comment]
Just reviewed and this is working as expected. Moving to close.
